### DISCUSSION TITLE
Display Tenant Name in NavBar

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#6787] [Help]
Please am using multitenancy in my serenity.
Please how can I get the Tenant name logged into on the NavBar for users to know which tenant they are connected to

**Comment by osmanaslancan** [comment]
if you add the multi-tenancy by following the serenity tutorial in this step https://serenity.is/docs/tutorials/multi_tenancy/filtering_users_by_tenantid you are putting TenantId to user cookie and adding an extension to retrive. 

To show the tenant name you should follow the same steps for tenant name. Then you can retrieve it using Context.User.GetTenantName() in a view.
